src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -33,6 +33,7 @@ import {getAppOptions} from '../../core/AppDataCntlr.js';
 import {HelpText} from './../../ui/HelpText.jsx';
 
 const vFileKey = LC.FG_FILE_FINDER;
+const DEFAULT_TITLE = 'Time Series Tool';
 /**
```

Comment:
It was set correctly when appTitle was undefined, but not when it was '', like in SUIT.
I did see `defaults.appTitle = 'Time Series Tool'` in IrsaViewer.js – it it's not used elsewhere, I would suggest to remove it, since it's the default title anyway. @ejoliet what do you think?